### PR TITLE
folding-at-home: Add version 7.6.21

### LINKED ISSUE #3694: [Request] Add Folding@Home
https://foldingathome.org

> **The Folding@home software runs while you do other things.**
> 
> While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.

### PR DESCRIPTION
**Pull request by schance995**
- Closes #3694

### CODE REVIEW

**Review by Ash258 — CHANGES_REQUESTED** [review]
https://github.com/lukesampson/scoop-extras/pull/4730/files

**Comment by schance995** [comment]
What is the correct way to write multiple licenses in the manifest?

The client is freeware but the control gui is GPL-3: <https://github.com/FoldingAtHome/fah-control>

**Comment by schance995** [comment]
Found the relevant information in the [docs](https://github.com/lukesampson/scoop/wiki/App-Manifests#optional-properties).
> If different files have different licenses, separate licenses with a comma (,).

> If there are both open-source and not open-source licenses, please list all non-open source licenses first

File: bucket/foldingathome.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "FAHControl.exe",
+            "FAHControl"
+        ],
+        [
+            "FAHViewer.exe",
+            "FAHViewer"
+        ]
+    ],
+    "persist": [
+        "work",
```
Comment (Ash258) [inline bucket/foldingathome.json:36]:
```suggestion
        "configs",
        "cores",
        "work",
        "logs",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
File persising do not work

A

**Review by Ash258 — CHANGES_REQUESTED** [review]
`git mv bucket/foldingathome.json bucket/folding-at-home.json`

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**